kubetest/e2e.go

```diff
@@ -623,10 +623,35 @@ func kubemarkTest(testArgs []string, dump, numNodes string) error {
 		})
 	}
 
+	if err := control.XmlWrap(&suite, "kubectl version", getKubectlVersion); err != nil {
+		return err
+	}
+
+	if err := control.XmlWrap(&suite, "IsUp", deploy.IsUp); err != nil {
+		return err
+	}
+
 	// Run tests on the kubemark cluster.
 	if err := control.XmlWrap(&suite, "Kubemark Test", func() error {
-		testArgs = util.SetFieldDefault(testArgs, "--ginkgo.focus", "starting\\s30\\pods")
-		return control.FinishRunning(exec.Command("./test/kubemark/run-e2e-tests.sh", testArgs...))
+		testArgs = append(
+			util.SetFieldDefault(testArgs, "--ginkgo.focus", "starting\\s30\\pods"),
+			"--e2e-verify-service-account=false",
+			"--dump-logs-on-failure=false",
+		)
+
+		cmd := exec.Command("./hack/ginkgo-e2e.sh", testArgs...)
+		cmd.Env = append(
+			os.Environ(),
+			"KUBERNETES_PROVIDER=kubemark",
+			"KUBE_CONFIG_FILE=config-default.sh",
+			fmt.Sprintf("KUBECONFIG=%s/kubernetes/test/kubemark/resources/kubeconfig.kubemark", os.Getenv("WORKSPACE")),
+			"E2E_MIN_STARTUP_PODS=0",
```

Comment:
I think you can get rid of this. It's a default - https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/test_context.go#L258

Reply:
ack